tools/mypy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# run mypy with project settings
+
+set -eu
+
+cd "$(dirname -- "${0}")/.."
```

Comment:
Same, i don't think this file helps.  I just do
```
mypy ipromise
```

Reply:
Currently, I recommended two paths, `mypy ipromise test`. It passes.

Reply:
Then just do `mypy ipromise tests`.  You can add that to the readme?

Reply:
Fixed.